### ISSUE TITLE
Warning: "Invalid prop `Component` supplied to `NavItem`" when using react-router Link

### ISSUE THREAD

**Issue by mikem-mup** [jxnblk/rebass#48]
It looks like the `Link` component class supplied by React-Router (v2.0.0) doesn't pass the `NavItem.propTypes.Component` test (it's not a `React.PropTypes.node`, it's a class).

My code:

```jsx
import { Link } from 'react-router';
// ...
<NavItem Component={Link} to='/'>Home</NavItem>
```

console warning:

```
Warning: Failed propType: Invalid prop `Component` supplied to `NavItem`, expected a ReactNode. Check the render method of `Root`.
```

It looks like the `Link` is rendered correctly, however, so I think it's just a matter of changing the `propTypes` rule to accept a component class?

**Comment by jxnblk** [comment]
Thanks! Seems like I was using the wrong prop type for that. I *think* it should be resolved in `0.2.4`, but let me know if you're still seeing warnings.

**Comment by mikem-mup** [comment]
Looks good, although I found the same issue on `LinkBlock` just now - should be identical fix.
